mautrix_signal/matrix.py

```diff
@@ -54,6 +54,21 @@ def __init__(self, bridge: "SignalBridge") -> None:
 
         super().__init__(bridge=bridge)
 
+    async def handle_invite(
```

Comment:
I don't think that you need this since `handle_puppet_invite` in https://github.com/mautrix/python/blob/master/mautrix/bridge/matrix.py#L306 should handle invites for you and call the portal function automatically, but I'm not 100% sure if that's how it works.

Reply:
This is for handling invites to users (rather than ghosts), so it is necessary currently